R/checkInputs.R

```diff
@@ -246,3 +246,31 @@ correctCdmFormat <- function(table, tableName) {
 
 
 }
+
+#' get occurrence information from cdm table
+#'
+#' @noRd
+#'
+getOccurrence <- function(cdm, tableName, start, end) {
+
+  dataTable <- cdm[[tableName]] |>
+    dplyr::select("person_id", start, end) |>
```

Comment:
start -> dplyr::all_of(start)
end -> dplyr::all_of(end)

Reply:
I forgot to delete this